lib/ClangImporter/ImportName.h

```diff
@@ -88,35 +76,47 @@ class ImportedName {
   EffectiveClangContext effectiveContext;
 
   struct Info {
+    /// For names that map Objective-C error handling conventions into
+    /// throwing Swift methods, describes how the mapping is performed.
+    ForeignErrorConvention::Info errorInfo;
+
+    /// For a declaration name that makes the declaration into an
+    /// instance member, the index of the "Self" parameter.
+    unsigned selfIndex;
+
+    /// For an initializer, the kind of initializer to import.
+    CtorInitializerKind initKind;
+
     /// The version of Swift this name corresponds to
-    ImportNameVersion version;
+    ImportNameVersion version : NumImportNameVersions;
+
+    /// What kind of accessor this name refers to, if any.
+    ImportedAccessorKind accessorKind : NumImportedAccessorKindBits;
 
     /// Whether this name was explicitly specified via a Clang
     /// swift_name attribute.
-    bool hasCustomName = false;
+    bool hasCustomName : 1;
```

Comment:
We avoid using `bool` with bitfields because (IIRC) MSVC treats them as signed and won't coalesce them with unsigned fields.